demo/app/script.ts

```diff
@@ -26,10 +26,18 @@ space.on('membersUpdate', (members) => {
 });
 
 /** Avoids issues with top-level await: an alternative fix is to change build target to es */
```

Comment:
This comment should be moved to the immediately invoked async function rather than above `init` I think